src/main/java/com/ait/lienzo/client/core/shape/TextTruncateWrapper.java

```diff
@@ -24,14 +24,12 @@
 /**
  * ITextWrapper implementation that truncates text and appends "..." if there is no space left.
  */
-@SuppressWarnings("Duplicates")
-public class TextTruncateWrapper extends TextNoWrap implements ITextWrapperWithBoundaries
+@SuppressWarnings("Duplicates") public class TextTruncateWrapper extends TextBoundsWrap
```

Comment:
some indentation / formatting issues on this class

Reply:
Hi @romartin I used a code style that I imported for lienzo, maybe it is outdated, can you send the one you use?

Reply:
TBH good point, because I just lost the code-style I'd in my local as I re-installed the OS.... can you please @hasys share it? IIRC we updated it somewhere guys, isn't it?

Reply:
Hi @romartin, we have no code-style for lienzo repositories. Probably, we should do it. But again the question is do we want to merge our changes back or not? If not we can use our KIE Group code style. If we have any plans around it - we should create and maintain new one, Lienzo specific.